examples/neighbors/plot_nca_classification.py

```diff
@@ -0,0 +1,87 @@
+"""
+============================================================================
+Comparing Nearest Neighbors and Neighborhood Components Analysis
```

Comment:
use with and without Neighborhood Components Analysis to avoid the confusion that NCA is itself a classifier

Reply:
You're right, thanks